Makefile

```
@@ -39,11 +39,8 @@ test-%: build-%
 test-%-cpp: build-%-cpp
 	TEST_USE_CPP=yes PYTHONPATH=$(wildcard ${_testbuilddir_abs}/lib*) $* test -v
 
-$(includedir):
-	mkdir -p $(includedir)
-
-py3c.pc: py3c.pc.in $(includedir)
-	sed -e's:@includedir@:$(realpath $(includedir)):' $< > $@
+py3c.pc: py3c.pc.in
+	sed -e's:@includedir@:$(includedir):' $< > $@
```

The `realpath` was an attempt to normalize the path (so e.g. if `prefix` ends with a `/`, you don't end up with `//` in the pkgconfig file). I guess `abspath` would work better for that:
```suggestion
	sed -e's:@includedir@:$($(abspath includedir)):' $< > $@
```

---

Thanks, just updated this to use `$(abspath $(includedir))` and amended the commit message.